KHLY2700/KHLY2700-IOC-01App/Db/keithley2700.db

```diff
@@ -373,7 +373,7 @@ record(calcout, "$(P)INDEX:START:UPDATE") {  # this needs to be triggered at the
   field(DESC, "Update start index after reads")
   field(INPA, "$(P)BUFF:NEXT") # May not work if record scans before read op completes
   field(CALC, "A")
-  field(OUT, "$(P)INDEX:START PP")
+  field(OUT, "$(P)INDEX:START")
```

Comment:
I would suggest using NPP and adding a comment, well done of finding the problem!